api/v1alpha/client_example.go

```diff
@@ -51,6 +51,24 @@ func main() {
 
 	for _, p := range podResp.Pods {
 		fmt.Printf("Pod %q is running\n", p.Id)
+
+		logsResp, err := c.GetLogs(context.Background(), &v1alpha.GetLogsRequest{
+			PodId: p.Id,
+		})
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(2)
```

Comment:
Dunno, this stupid error numbering. I guess it should be 3, and then 4, and then 5. I can just get rid of those like we did with init.go some time ago